tests/ui/suggestions/missing-lifetime-in-assoc-const-type.generic_const_items.stderr

```diff
@@ -20,16 +15,11 @@ help: consider introducing a named lifetime parameter
 LL |     const B<'a>: S<'a> = S { s: &() };
    |            ++++   ++++
 
-error[E0106]: missing lifetime specifier
+error[E0637]: `'_` cannot be used here
   --> $DIR/missing-lifetime-in-assoc-const-type.rs:8:15
    |
 LL |     const C: &'_ str = "";
-   |               ^^ expected named lifetime parameter
-   |
-help: consider introducing a named lifetime parameter
-   |
-LL |     const C<'a>: &'a str = "";
-   |            ++++   ~~
```

Comment:
I guess I could probably recover this suggestion by inverting the order of the lifetime ribs... Doesn't seem particularly useful, though.

Reply:
If we end up deciding on dropping those suggestion diagnostics (personally I'm fine either way, also don't know how useful they are, mainly they're consistent with other diagnostics), then you can get rid of the revisions of this test as well as `LifetimeBinderKind::ConstItem` (→ `Item`) in `late/diagnostics.rs` since it will become unreachable.